src/main/java/is/ru/tictactoe/Cli.java

```diff
@@ -69,11 +71,23 @@ private void greeting() {
 	// Translate index to x,y coordinates
 	private coordinates getCellCords(int index) {
 		coordinates cord = new coordinates();
-		cord.x = (index % 3);
-		cord.y = (index / 3);
+	
+		cord.x = setCordx(index);	
+		cord.y = setCordy(index);	
+	//	cord.x = (index % 3);
+	//	cord.y = (index / 3);
 		return cord;
 	}
 
+	public int setCordx(int index) {
+		return (index % 3);
+	}
+
+	public int setCordy(int index) {
```

Comment:
I feel that the names of these two functions are wrong.  The 'set' part indicates that they're setting something, but they're completely side effect free.

Maybe they should simply be called `cordX` and `cordY`?